samples/event_manager/README.rst

```diff
@@ -28,7 +28,7 @@ Controller (``controller.c``):
 
 Statistics (``stats.c``):
   This module waits for measurement events from the sensor.
-  Every time a measurement event is received, the module calculates simple statistics about one of the measurement values that are transmitted as part of the event and logs them.
+  The module calculates simple statistics about one of the measurement values that are transmitted as part of the event and logs them.
```

Comment:
```suggestion
The module calculates and logs basic statistics about one of the measurement values that are transmitted as part of the event.
```
`one of the values` -- maybe it is worth mentioning which one?

Reply:
This is just a sample event. The values are called `value1`, `value2` and `value3`. I think it's not needed here.